### ISSUE TITLE
Changing strokeWidth

### ISSUE THREAD

**Issue by oddgrd** [asotog/fabricjs-react#4]
I am making an app to move circles, rects etc. over holds on a  climbing board (the board is the background img). I would like to change the strokeWidth of objects before adding them, like you can change the color before adding them. As far as I have found, this is not supported by fabricjs-react. I have been able to change the property by getting the object/objects and changing it on each object, but this only applies on resizing the objects with mouse-drag.

Change the strokewidth of all objects (only applies on mouse-drag resize of object):
`
useEffect(() => {
    editor?.canvas.getObjects().map((obj) => (obj.strokeWidth = strokeWidth));
  });
`
This is my first time working with canvas, fabric and fabricjs-react. Any tips are greatly appreciated!

**Comment by asotog** [comment]
hi @oddgrd  can you try calling renderAll on canvas you modify the stroke ?  like
```
editor?.canvas.renderAll();
```

**Comment by oddgrd** [comment]
Thank you for your suggestion. I tried using renderAll(), unfortunately it didn't seem to change anything. I was only able to apply the strokewidth change on seemingly arbitrary events, like rescaling and zooming. Luckily I managed to google the correct phrase today, and found this: http://fabricjs.com/fabric-gotchas. My issue is explained in the second-to-last section.

After reading this, those events don't seem so arbitrary anymore, and even though I am not able to set the strokewidth before adding objects, it should appear to work like that for the user. Thanks for making this package, it has saved me a lot of time and headache!

**Comment by asotog** [comment]
thanks for pointing out this, we'll try to create a wrapper for object modifications, so develop does not have to call the fabric render API stuff. but does it work for you to make changes first and then calling the `renderAll`?

**Comment by oddgrd** [comment]
Yes, it now works as I intended. On creating a circle, for example, it is instantly changed to the strokeWidth I set it too. To be spesific: as I add a circle, I also call a function that updates state and causes a component update, on component update the useEffect hook gets all objects and sets their strokewidth, and then I call renderAll(). (to optimize I will probably change it to only affect the last added object). If I don't call renderAll() here, it only changes width on the objects after I select them.
